packages/google_workspace/data_stream/user_accounts/elasticsearch/ingest_pipeline/default.yml

```
@@ -94,6 +94,30 @@ processors:
       field: json.events.type
       target_field: google_workspace.event.type
       ignore_missing: true
+  - script:
+      decription: |
+        Moves all objects into the 'parameters' array into their own keyed object under 'google_workspace.user_accounts.*'.
```

```suggestion
        Moves all objects in the 'parameters' array into their own keyed object under 'google_workspace.user_accounts.*'.
```

Did you mean "in" or "from"?